pkgs/tools/misc/android-tools/default.nix

```diff
@@ -16,6 +16,19 @@ stdenv.mkDerivation rec {
     hash = "sha256-YCNOy8oZoXp+L0akWBlg1kW3xVuHDZJKIUlMdqb1SOw=";
   };
 
+  patches = [
+    # Fix building with newer protobuf versions.
+    (fetchpatch {
+      url = "https://gitlab.archlinux.org/archlinux/packaging/packages/android-tools/-/raw/main/protobuf-23.patch";
+      hash = "sha256-KznGgZdYT6e5wG3gtfJ6i93bYfp/JFygLW/ZzvXUA0Y=";
+    })
```

Comment:
- using `main` seems risky, as the patch probably will be deleted by them on the branch at some point; better fetch from a particular commit
- I'd use plain `fetchurl` instead of `fetchpatch`.  It's not a dynamically generated patch, so I see little use in the normalization, and plain hashing without touching contents is less confusing for people.

Reply:
Both done.